lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -472,8 +472,11 @@ sub stat_sensitivity_exact {
     my $results = $self->get_rawstats();
 
     my $total_true = $self->stat_total_true_positive_exact;
-    if ($total_true == 0) {
-        return 0;
+    {
+        no warnings;
```

Comment:
For that case you could use:
```perl
no warnings qw(numeric);
```

`nan` (also `NaN`) is a numeric value, but it does not equal zero.  `NA` is not a numeric value, but it does equal zero.  If the set of non-numerical results could be any string, you cannot know if it will be equal to zero.  (For example, `2FA` is not numeric, but equals `2`.)   It might be better to use `Scalar::Util`'s `looks_like_number` and handle non-numbers specifically.